.gitignore

```diff
@@ -61,7 +61,7 @@ typings/
 .next
 
 # rollup build output
-es/*
+esm/*
 lib/*
```

Comment:
I think since there are two full trees that this is more appropriate -- using `dist` seems to be more when there are just individual file exports. Going through some node_modules in projects, it seems pretty split that way. You have a strong opinion on it or a link to some best practice guide about it? (I'm open to changing it)

Reply:
No strong opinion -- just noticed that you have to modify .gitignore, gulp clean, and add gulp build steps any time you want to add a new bundled module vs just having to add a new build step. Just one less thing to have to remember to do, although, not exactly the most critical thing in the world to forget.

Reply:
Okay, I'm going to leave it as is